boa_engine/src/bytecompiler/jump_control.rs

```diff
@@ -0,0 +1,344 @@
+use crate::{
```

Comment:
It would be good to see top level documentation for this, doesn't have to be the level of [this](https://github.com/boa-dev/boa/blob/main/boa_engine/src/environments/mod.rs#L1-L25) but similar.